server/routes/customers.js

```diff
@@ -50,7 +50,8 @@ router.post('/', (req, res) => {
     })
       .save()
       .then(customer => {
-        return res.json(customer);
+        const newCustomer = customer.toJSON()
```

Comment:
So there's a `res.json` function you can use to send back javascript objects without having to stringify them first